localisation/00_wc_sad.csv

```diff
@@ -411,7 +411,7 @@ EVTOPTA_WCCLS.10060;I will do my best.;;Ich werde mein Bestes geben.;;;;;;;;;;;x
 EVTOPTA_WCCLS.10061;I will depart immediately.;;Ich werde sofort abreisen.;;;;;;;;;;;x
 EVTOPTB_WCCLS.10061;I already bear his blessing.;;Ich trage bereits seinen Segen.;;;;;;;;;;;x
-EVTDESC_WCCLS.10062;You have gained blessing of Malorne and will now face a final test. You will undergo a ritual to relive Malornes memories from War of the Ancients and face Archimonde the Defiler.;;Ihr habt den Segen von Malorne erhalten und werdet nun einer letzten Pr�fung unterzogen. Ihr werdet ein Ritual durchf�hren, um Malornes Erinnerungen an den Krieg der Uralten wieder aufleben zu lassen und Archimonde dem Verderber gegen�ber zu treten.;;;;;;;;;;;x
+EVTDESC_WCCLS.10062;You have gained blessing of Malorne and will now face a final test. You will undergo a ritual to relive Malornes memories from War of the Ancients and face Archimonde the Defiler.;;Ihr habt den Segen von Malorne erhalten und werdet nun einer letzten Pr�fung unterzogen. Ihr werdet ein Ritual durchf�hren, um Malornes Erinnerungen an den Krieg der Ahnen wieder aufleben zu lassen und Archimonde dem Verderber gegen�ber zu treten.;;;;;;;;;;;x
 EVTOPTA_WCCLS.10062;I will not fail him.;;Ich werde ihn nicht entt�uschen.;;;;;;;;;;;x
 EVTDESC_WCCLS.10063;The ritual has begun. As the world swirls around you the vision blurs and you start to hear weird sounds. You see the Defiler before you. Overcome with rage and fury you charge at him with your antlers. The ground trembles under your hooves as you sweep demons aside. You reach your enemy and he barely dodges your attack, saving himself from being impaled. You exhange blows for hours and you you're getting more and more tired with each passing moment. Archimonde notices it and grabs your neck with his demonic strength. You struggle to free yourself but to no avail. He snaps your neck and as you leave the vision the last thing you hear is powerful roar of furious Ysera.;;Das Ritual hat begonnen. W�hrend die Welt um Euch herum wirbelt, verschwimmt Eure Sicht und Ihr beginnt, seltsame Ger�usche zu h�ren. Ihr seht den Verderber vor Euch. �berw�ltigt von Zorn und Raserei, st�rmt lhr mit Eurem Geweih auf ihn zu. Der Boden zittert unter Euren Hufen, w�hrend Ihr D�monen beiseite fegt. Ihr erreicht Euren Feind und er weicht Eurem Angriff kaum aus, wodurch er sich davor bewahrt, aufgespie�t zu werden. Stundenlang tauscht ihr Schl�ge aus und Ihr werdet immer ersch�pfter. Archimonde bemerkt dies und packt Euren Hals mit seiner d�monischen St�rke. Ihr k�mpft darum, Euch zu befreien, aber ohne Erfolg. Er bricht Euch den Hals und als Ihr die Vision verlasst, ist das Letzte, was Ihr h�rt, das machtvolle Br�llen der w�tenden Ysera.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCCLS.10063;The ritual has begun. As the world swirls around you the vision blurs and you start to hear weird sounds. You see the Defiler before you. Overcome with rage and fury you charge at him with your antlers. The ground trembles under your hooves as you sweep demons aside. You reach your enemy and he barely dodges your attack, saving himself from being impaled. You exchange blows for hours, but you are getting more and more tired with each passing moment. Archimonde notices it and grabs your neck with his demonic strength. You struggle to free yourself but to no avail. He snaps your neck and as you leave the vision the last thing you hear is the furious roar of Ysera.;;Das Ritual hat begonnen. W�hrend die Welt um Euch herum wirbelt, verschwimmt Eure Sicht und Ihr beginnt, seltsame Ger�usche zu h�ren. Ihr seht den Verderber vor Euch. �berw�ltigt von Zorn und Raserei, st�rmt lhr mit Eurem Geweih auf ihn zu. Der Boden zittert unter Euren Hufen, w�hrend Ihr D�monen beiseite fegt. Ihr erreicht Euren Feind und er weicht Eurem Angriff kaum aus, wodurch er sich davor bewahrt, aufgespie�t zu werden. Stundenlang tauscht ihr Schl�ge aus und Ihr werdet immer ersch�pfter. Archimonde bemerkt dies und packt Euren Hals mit seiner d�monischen St�rke. Ihr k�mpft darum, Euch zu befreien, aber ohne Erfolg. Er bricht Euch den Hals und als Ihr die Vision verlasst, ist das Letzte, was Ihr h�rt, das machtvolle Br�llen der w�tenden Ysera.;;;;;;;;;;;x
```

Reply:
I think, exchanging blows for hours doesn't contradict getting tired. Would keep the original version.

Reply:
At least deal with the you you're thing then. That was my chief complaint.

Reply:
I totally didn't see that earlier xD Now I did